app/main.py

```diff
@@ -1,6 +1,22 @@
+import json
+import os
+from dotenv import load_dotenv
+
+import requests
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+API_KEY = os.environ["API_KEY"]
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    url = BASE_URL + f"key={API_KEY}&q={CITY}"
+    res = requests.get(url)
```

Comment:
The API request lacks error handling. It's important to handle potential errors such as network issues or invalid responses. Consider checking the response status code and managing exceptions appropriately.